src/objects/tokens/mirage.js

```diff
@@ -0,0 +1,85 @@
+const { refObject, Rotator } = require("../../wrapper/api");
+const { System, Planet } = require("../../lib/system/system");
+const { Broadcast } = require("../../lib/broadcast");
+const Vector = require("../../mock/mock-vector");
```

Comment:
careful, VSCode sometimes "helps" by giving you the wrong require.. this should be in the "wrapper/api" set.